scripts/build_packages.sh

```diff
@@ -55,8 +55,6 @@ if [ "${PKG_ROOT}" = "" ]; then
     PKG_ROOT=${HOME}/node_pkg
 fi
 
-rm -rf ${PKG_ROOT}
```

Comment:
I think we may have some assumptions with our scripting that there is only one build version in `PKG_ROOT`.  I'm not sure we'd want to change this behavior without testing.

Reply:
We do assume that there can be multiple builds under PKG_ROOT, so we don't want to delete what's already there.